src/ModelContextProtocol/Utils/Json/CustomizableJsonStringEnumConverter.cs

```diff
@@ -21,7 +22,8 @@ namespace System.Text.Json.Serialization;
 /// 9.x support for custom enum member naming. It will be replaced by the built-in functionality
 /// once .NET 9 is fully adopted.
 /// </remarks>
-internal sealed class CustomizableJsonStringEnumConverter<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields)] TEnum> :
+[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
Needed when source generating protocol types in user code.

Reply:
Rhetorical question. I don't think we have any. If necessary we can keep this type even if no longer being used by the models.

Reply:
At a minimum we should change the namespace. And possibly a better name. And hopefully find a way to remove it later.

Reply:
I think the name current is acceptable. What sort of namespace do you have in mind?

Reply:
Moved it under `ModelContextProtocol.Utils.Json`